### ISSUE TITLE
!jobs - List jobs in CareerHub

### ISSUE THREAD

**Issue by lewisjb** [UQComputingSociety/uqcs-hubot#81]
Similar to !events.

Probably scrape off https://www.careerhub.uq.edu.au/students/jobs/search?query=&typeofwork=-1&location=&country=Any&occupation=916&residency=

For more than 10 (probably PM them at that point) most likely want to scrape `"https://www.careerhub.uq.edu.au/students/jobs/search?query=&typeofwork=-1&location=&country=Any&occupation=916&residency=&page=1&take=" + amount`

Possibly add caching

**Comment by nickl93** [comment]
Could we do internship application (for tech companies) dates too?

**Comment by lewisjb** [comment]
Good idea, but will need a way to keep track of them, maybe a google calendar?

Day suggested implementing the Seek API aswell which seems like a good idea
